lib/testRun.js

```diff
@@ -63,6 +63,11 @@ module.exports = function() {
       if (text.match(/^[a-z0-9]+$/)) {
         annotation.key = text;
       } else {
+        var active = parseAnnotationValue(text.toLowerCase(), /active\=[\"\']?([1|0|true|false|yes|no])[\"\']?/);
```

Comment:
To my knowledge, this: `([1|0|true|false|yes|no])`...

Is equivalent to this: `([1|0truefalsyn])` which matches one (and only one) of these characters.

The `[]` within the parentheses should be removed.

Additionally, if you're trying to re-use the same format we accept elsewhere, you should add `y|n|t|f`.